stable/kibana/values.yaml

```diff
@@ -16,6 +16,19 @@ env:
   # LOGGING_VERBOSE: "true"
   # SERVER_DEFAULTROUTE: "/app/kibana"
 
+configFiles:
+  kibana.yml:
+    ## Default Kibana configuration from kibana-docker.
+    server.name: kibana
+    server.host: "0"
+
+    ## Custom config properties below
+    ## Ref: https://www.elastic.co/guide/en/kibana/current/settings.html 
+    # elasticsearch.url: http://elasticsearch-elasticsearch-client:9200
```

Comment:
default kibana.yml on `6.1.3` includes this line uncommented. in the previous chart modality we relied on env to override it, but probably best to uncomment and move under the default section. 

```
elasticsearch.url: http://elasticsearch:9200
```

^^^ is the default on 6.1.3 at least (haven't checked other versions)